image/structure/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2016 The Linux Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package structure defines stucts for image-spec content.
+package structure
```

Comment:
On Mon, Jun 20, 2016 at 06:01:08AM -0700, Sergiusz Urbaniak wrote:
> > +// Package structure defines stucts for image-spec content.
> > +package structure
> 
> There's merely on struct defined here. Since we wanted the oci-image
> repo to include only schemas, I'd be careful exposing any structs.

Two structures: Descriptor and ImageLayoutVersion.  And #147 is
planning on exporting auto-generated versions, in which case we can
drop these hand-coded structures and adjust the cas/ref engines to use
those.  Is “not exposing structures for handling JSON defined in this
spec” on the table?  I think #147 is “should we autogenerate these or
write them by hand?” and not “should we expose these at all?”.